patches/chrome-installer-mac-signing-signing.py.patch

```diff
@@ -27,3 +27,22 @@ index b56853e5e2649328a81a2de20228de5abda08f11..b0e4a28355f410efb2105273f423d4f3
          'libEGL.dylib',
          'libGLESv2.dylib',
          'libswiftshader_libEGL.dylib',
+@@ -143,7 +149,7 @@ def sign_part(paths, config, part):
+         part: The |model.CodeSignedProduct| to sign. The product's |path| must
+             be in |paths.work|.
+     """
+-    command = ['codesign', '--sign', config.identity]
++    command = ['codesign', '--force', '--sign', config.identity]
```

Comment:
`--force` is needed to signing again after inserting sig file into framework bundle.